Makefile

```diff
@@ -46,4 +46,14 @@ check_trufflehog:
 trufflehog: check_trufflehog
 	trufflehog --regex --rules .trufflehog.json --entropy=False file:///$$azul_home
 
-.PHONY: all hello terraform deploy subscribe everything reindex clean test travis integration_test trufflehog check_trufflehog
+demolish: check_branch_personal
+	$(MAKE) unsubscribe
+	$(MAKE) -C terraform destroy
+	$(MAKE) -C lambdas delete
+
+force_demolish: check_branch_personal
+	$(MAKE) unsubscribe
+	$(MAKE) -C terraform auto_destroy
+	$(MAKE) -C lambdas delete
+
+.PHONY: all hello terraform deploy subscribe everything reindex clean test travis integration_test trufflehog check_trufflehog delete force_delete
```

Comment:
You need to add `demolish ` and `force_demolish ` phony targets. And is `force_delete` a target and should it still be in `PHONY`?

Reply:
Thanks, good catch.